File: src/com/anathema_roguelike/main/ui/charactercreation/attributeselectors/PointBuy.java
package com.anathema_roguelike.main.ui.charactercreation.attributeselectors;

import com.anathema_roguelike.characters.Player;

public class PointBuy extends AttributeSelector {

	@Override
	public void selectScores(Player player) {
		new PointBuyScreen("Set your Ability Scores", player).run();
	}
}

File: src/com/anathema_roguelike/main/ui/charactercreation/attributeselectors/PointBuy.scala
package com.anathema_roguelike
package main.ui.charactercreation.attributeselectors

import com.anathema_roguelike.entities.characters.player.Player

class PointBuy extends AttributeSelector {
  def selectScores(player: Player): Unit = {
    new PointBuyScreen("Set your Ability Scores", player).run
  }
}
